### PR TITLE
Add localization to plugin

### PR DESCRIPTION
**Pull request by pedro-mendonca**
Code based on GitHub Updater

### CODE REVIEW

**Comment by szepeviktor** [comment]
Thank you very much.
This is the first PR in my life.

**Comment by pedro-mendonca** [comment]
:)
Well, I though it deserved to look like its parent plugin :+1:

File: languages/github-link-pt_PT.po [hunk 1]
```diff
@@ -0,0 +1,35 @@
+msgid ""
+msgstr ""
+"Project-Id-Version: GitHub Link\n"
```
Comment (szepeviktor) [inline languages/github-link-pt_PT.po:3]: What are these newlines? `\n`

Comment (szepeviktor) [inline languages/github-link-pt_PT.po:3]: Got it: http://pology.nedohodnik.net/doc/user/en_US/ch-poformat.html#sec-poheader

Comment (pedro-mendonca) [inline languages/github-link-pt_PT.po:3]: Wich ones? Whe whole header is generated by Poedit, the tool I use to generate .pot / .po files

Comment (szepeviktor) [inline languages/github-link-pt_PT.po:3]: I meant the newline at the end of the strings.

Comment (pedro-mendonca) [inline languages/github-link-pt_PT.po:3]: Poedit always closes every keyline in header like that.
If you manually edit it out, Poedit wont even recognize it as a filed and return an error.
I think is related to Unix and all Unix-like systems, \n is the code for end-of-line.

**Pull request merged**